packages/prettier-config/lib/index.js

```diff
@@ -1,4 +1,13 @@
-module.exports = {
+/** @typedef {import('prettier').Options} PrettierOptions */
+
+/**
+ * @typedef WPPrettierOptions
+ *
+ * @property {boolean} [parenSpacing=true] Insert spaces inside parentheses.
+ */
+
+/** @type {PrettierOptions & WPPrettierOptions} */
```

Comment:
A union - cool 👍

Reply:
Unfortunately, `jsdoc/valid-types` doesn't like the union. We'll need eslint-disable here.

Reply:
> Unfortunately, `jsdoc/valid-types` doesn't like the union. We'll need eslint-disable here.

Done in a450cecb99.

Related: https://github.com/gajus/eslint-plugin-jsdoc/issues/145